docs/Get started/FAQ.md

```diff
@@ -614,7 +614,7 @@ Yes! Please sign up for the [Fleet Cloud Beta](https://kqphpqst851.typeform.com/
 
 ### What MySQL versions are supported?
 
-Fleet is tested with MySQL 5.7.21 and 8.0.28. Newer versions of MySQL 5.7 and MySQL 8 typically work well. AWS Aurora requires at least version 2.10.0. Please avoid using MariaDB or other MySQL variants that are not officially supported. Compatibility issues have been identified with MySQL variants, and these may not be addressed in future Fleet releases.
+Fleet is tested with MySQL 8.0.36. Newer versions of MySQL 8 typically work well. AWS Aurora requires at least version 2.10.0. Please avoid using MariaDB or other MySQL variants that are not officially supported. Compatibility issues have been identified with MySQL variants, and these may not be addressed in future Fleet releases.
```

Comment:
With the 8.0 upgrade, Aurora should be spec'd as "version 3" in general, or >= 3.07.0 specifically as that version is the first one with 8.0.36 compat (current version as of writing this is 3.07.1, so you can't go any later than 8.0.36).

Source: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraMySQLReleaseNotes/AuroraMySQL.Updates.30Updates.html

This is another reason for not going past 8.0.36 at the moment.

Reply:
Thanks for catching this @iansltx. I've opened https://github.com/fleetdm/fleet/pull/21130.